.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--
+Describe your changes in detail.
+For contribution process guide, look into CONTRIBUTING.md
```

Comment:
please make CONTRIBUTING.md a link

Reply:
heh, this template is technically a `.md` file, but it will be displayed as a plain text, so that wouldn't look good, I think

Reply:
You could use this link: https://github.com/oneapi-src/unified-memory-framework/blob/main/CONTRIBUTING.md
It  displays formatted properly.

Reply:
I'm just saying the content we're putting here in the template will be displayed plain text, so I'm not sure if providing [md links](link) or whole link is such a good idea, but maybe I'm wrong 😉

Reply:
I think "look intro CONTRIBUTING.md in the main directory" or "top level" would be enough

Reply:
sounds good, done.